src/IotHub/IotHub/IotHub/GetAzureRmIotHubKey.cs

```diff
@@ -27,9 +27,22 @@ namespace Microsoft.Azure.Commands.Management.IotHub
     [OutputType(typeof(PSSharedAccessSignatureAuthorizationRule))]
```

Comment:
@anusapan same comment about adding the `DefaultParameterSetName` above

Reply:
Done